### PR TITLE
Bring back strongly typed mappers

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change

D

### CODE REVIEW

File: src/Controls/src/Core/HandlerImpl/Button/Button.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public partial class Button
 		public static IPropertyMapper<IButton, ButtonHandler> ControlsButtonMapper = new PropertyMapper<Button, ButtonHandler>(ButtonHandler.Mapper)
 		{
 			[nameof(ContentLayout)] = MapContentLayout,
-#if IOS
+#if __IOS__
```
Comment (mattleibow) [inline src/Controls/src/Core/HandlerImpl/Button/Button.cs:18]: What is the reason for this one? Is there a difference?

File: src/Controls/src/Core/HandlerImpl/Button/Button.iOS.cs [hunk 2]
```diff
@@ -9,10 +9,8 @@ namespace Microsoft.Maui.Controls
 {
 	public partial class Button
 	{
-		private static void MapPadding(IButtonHandler handler, Button button)
```
Comment (mattleibow) [inline src/Controls/src/Core/HandlerImpl/Button/Button.iOS.cs:12]: Why are all the padding ones private?

File: src/Controls/src/Core/HandlerImpl/Label/Label.cs [hunk 3]
```diff
@@ -18,7 +18,7 @@ public partial class Label
 #if ANDROID
 			[nameof(TextColor)] = MapTextColor,
 #endif
-#if IOS
+#if __IOS__
```
Comment (mattleibow) [inline src/Controls/src/Core/HandlerImpl/Label/Label.cs:21]: Another change

File: src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Tizen.cs [hunk 4]
```diff
@@ -10,58 +10,58 @@ public partial class Toolbar
 		IPlatformViewHandler? _nativeTitleViewHandler;
 		MauiToolbar PlatformView => Handler?.PlatformView as MauiToolbar ?? throw new InvalidOperationException("Native View not set");
 
-		public static void MapBarTextColor(IToolbarHandler handler, Toolbar toolbar)
```
Comment (mattleibow) [inline src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Tizen.cs:13]: I think tizen is also missing the chunk from the other platforms for the interfaces.

**Review by mattleibow — COMMENTED** [review]
almost there!

File: src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Tizen.cs (hunk 4 above)
Comment (PureWeen) [reply src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Tizen.cs:13]: should be added back now

File: src/Controls/src/Core/HandlerImpl/Label/Label.cs (hunk 3 above)
Comment (PureWeen) [reply src/Controls/src/Core/HandlerImpl/Label/Label.cs]: done

File: src/Controls/src/Core/HandlerImpl/Button/Button.iOS.cs (hunk 2 above)
Comment (PureWeen) [reply src/Controls/src/Core/HandlerImpl/Button/Button.iOS.cs:12]: So we can avoid this type of PR for NET8 :-)

File: src/Controls/src/Core/HandlerImpl/Button/Button.cs (hunk 1 above)
Comment (PureWeen) [reply src/Controls/src/Core/HandlerImpl/Button/Button.cs]: reverted

**Comment by mattleibow** [comment]
Merged into https://github.com/dotnet/maui/pull/10410

**Pull request closed without merging**